package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "taverna_web_client",
   "version": "1.0.0",
-  "description": "\"Meal service food for whatever it is we can use it for",
+  "description": "The admin interface for Tarvena app",
```

Comment:
@andela-kanyanwu  I'd love you weigh in on this one. Is this the expected description?